manifests/machineconfig.crd.yaml

```diff
@@ -39,205 +41,401 @@ spec:
+                            items:
+                              type: string
+                          spares:
+                            type: integer
                 systemd:
-                  description: "systemd describes the desired state of the systemd units"
                   type: object
                   properties:
                     units:
-                      description: "units is a list of units to be configured"
                       type: array
                       items:
-                        description: "items describes unit configuration"
                         type: object
                         properties:
-                          name:
-                            description: "name is the name of the unit. This must be suffixed with a valid unit type (e.g. 'thing.service')"
-                            type: string
-                          enabled:
-                            description: "enabled describes whether or not the service shall be enabled. When true, the service is enabled. When false, the service is disabled. When omitted, the service is unmodified. In order for this to have any effect, the unit must have an install section"
-                            type: boolean
-                          mask:
-                            description: "mask describes whether or not the service shall be masked. When true, the service is masked by symlinking it to /dev/null"
-                            type: boolean
                           contents:
-                            description: "contents is the contents of the unit"
                             type: string
                           dropins:
-                            description: "dropins is the list of drop-ins for the unit"
                             type: array
                             items:
-                              description: "items describes unit dropin"
                               type: object
                               properties:
                                 contents:
-                                  description: "contents is the contents of the drop-in"
                                   type: string
                                 name:
-                                  description: "name is the name of the drop-in. This must be suffixed with '.conf'."
                                   type: string
-          kargs:
-            description: "kargs contains a list of kernel arguments to be added"
-            type: array
-            items:
-              description: "kernel argument"
+                          enable:
+                            type: boolean
+                          enabled:
+                            type: boolean
+                          mask:
+                            type: boolean
+                          name:
+                            type: string
+            fips:
+              type: boolean
+            kernelArguments:
```

Comment:
Similar comment what Erica already mentioned. We should preserve description of these too